common/app/views/fragments/items/facia_cards/contentCard.scala.html

```diff
@@ -183,7 +183,7 @@
                                 classes = Seq("responsive-img "),
                                 widths = item.mediaWidthsByBreakpoint,
                                 maybePath = Some(imageElement.url),
-                                maybeSrc = if(containerIndex == 0 && index < 4) Some(imageElement.url) else None,
+                                maybeSrc = if(containerIndex == 0 && index < 9) Some(imageElement.url) else None,
```

Comment:
Looking at this code again, I think the `index < 4` should have been kept as-is.

See fix in #25497